file_sync_only/Tiltfile

```diff
@@ -18,41 +32,53 @@ def file_sync_only(image='', manifests=[], deps=["."], live_update=[]):
         skips_local_docker=True,
         live_update=live_update
     )
-    k8s_yaml(manifests)
-    _first_sync_from_liveupdate(deployed_image, live_update)
-
-# get current tags from image setting from first arg in file_sync_only() or in manifests
-def _get_current_tag(image='', manifests=[]):
-    p = image.split(':')
-    if len(p) == 2:
-        return p[0], p[1]
-
-    tags = []
-
-    for m in manifests:
-        output = str(local(r"grep 'image: %s' %s | cut -d ':' -f 3 | uniq || true" % (image, m))).strip()
-        if (output != ''):
-            tags = tags + output.split("\n")
-
-    if len(tags) == 0:
-        tags = ["latest"]
-    elif len(tags) > 1:
-        fail("found multiple image tags for %s in manifests %s" % (image, manifests))
-
-    return image, tags[0]
+    # k8s_yaml(manifests, allow_duplicates=allow_duplicates)
```

Comment:
did you intend to comment this out? i think this is why CI is failing.

Reply:
@nicks 
sorry for late reply.
I fixed code and tests.